stdeb/pypi_simple.py

```diff
@@ -0,0 +1,46 @@
+import re
+import requests
+import stdeb
+
+USER_AGENT = "pypi-install/%s ( https://github.com/astraw/stdeb )" % \
+    stdeb.__version__
+
+
+def normalize_package_name(package_name):
+    return re.sub(r"[-_.]+", "-", package_name).lower()
+
+
+class PyPIClient(object):
+    def __init__(self, pypi_url="https://pypi.org", user_agent=USER_AGENT):
+        self.pypi_url = pypi_url
+        self._http_session = requests.Session()
+        self._http_session.headers["User-Agent"] = user_agent
+
+    def release_versions(self, package_name, current_only=False):
+        package_name = normalize_package_name(package_name)
+        if current_only:
+            response = self._http_session.get("%s/pypi/%s/json" % (self.pypi_url, package_name))
+            data = response.json()
+            return data['info']['version']
```

Comment:
I believe that all of the other string quotes in this file are double quotes. It would be nice to be consistent.

That said, the other file you're changing here seems to use single quotes. Your call.

```suggestion
            return data["info"]["version"]
```

Reply:
Thanks. I think the `ruff` style which I'm exploring for this project uses double quotes ubiquitously and this project currently uses a mix of quote characters.

Fixed in 2d0a0898a57dc4fab13c63b41216fbf4582257a9